PriceKataCalc/Program.cs

```diff
@@ -1,2 +1,31 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿using System;
+namespace Kata
+{
+    class priceCalc
+    {
+        public static void Main()
+        {
+            product myProduct = new("ball", 20405, 15.1567,0.20);
+            Console.WriteLine($"Product price reported as {myProduct.price} before tax and {myProduct.price+myProduct.calcTax()} after {myProduct.tax*100}% tax. ");
+
+        }
+       
+    }
+    class product{
```

Comment:
Usually it is the convection to make new classes in a new files and the class names is in pascal case